appcoinsRewards/src/main/java/com/appcoins/wallet/appcoins/rewards/AppcoinsRewards.kt

```diff
@@ -18,7 +19,8 @@ class AppcoinsRewards(
     private val cache: Repository<String, Transaction>,
     private val scheduler: Scheduler,
     private val billing: Billing,
-    private val errorMapper: ErrorMapper) {
+    private val errorMapper: ErrorMapper,
+    private val partnerAddressService: AddressService) {
```

Comment:
Why do we have the need of this dependency?why don't we send the wallet address as a function argument?